### PR TITLE
feat(tax costs): add annual income and filing status as required fields

### PR DESCRIPTION
**Pull request by kvhswong**
- Supports: HOME-532

### CODE REVIEW

**Comment by kvhswong** [comment]
Don't merge yet, need to resolve build issues with platform and consumerui. These all need to be merged together otherwise the onboarding flow will fail.

**Comment by zrangerz** [comment]
I’ll add +1s to whatever I see working and you can handle merges

--
Steve Mays
Sent with Airmail

File: api/v3/accounts/schema/accounts.req.json [hunk 1]
```diff
@@ -16,6 +16,11 @@
       "description": "The signer's full name.",
       "type": "string"
     },
+    "taxFilingStatus": {
+      "type": "string",
+      "enum": ["SINGLE", "MARRIED_FILING_JOINTLY", "MARRIED_FILING_SEPARATELY", "HEAD_OF_HOUSEHOLD", "QUALIFYING_WIDOWER"]
+    },
+    "annualTaxableIncome": {"type": "number"},
```
Comment (abuecker) [inline api/v3/accounts/schema/accounts.req.json]: Probably want to assign a format to it so the mocks are reasonable like:

https://github.com/trizic/api-docs-consumer/blob/master/server/formats.js#L111-L113

```
{
  "type": "number",
  "format": "amount"
}
```

Comment (kvhswong) [inline api/v3/accounts/schema/accounts.req.json]: Thanks for pointing me to the formats file. I've updated the commit to include the amount format.

**Comment by abuecker** [comment]
👍

**Pull request merged**